CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to SeedLang
+
+Thanks for taking the time to contribute!
+
+## Design decisions
+
+There are lots of trade-offs to made when building a new programming
+environment. We will document every significant design decision in the `design`
+dir. Please create a new issue labeled with `design` if you find anything
+undocumented or propose an update for an existing design.
+
+## Submitting changes
+
+For the main branch, all commits must be submitted via a pull request with at
+least one approving reviews from the core team.
+
+For the developers who have write-access to the repository, please consider the
+following branch naming conventions before submitting a pull request:
+
+ * Release branches: `release_<version>`
+ * Personal working branches: `wip_<your-github-id>` or `wip_<your-github-id>_<sequence_no>`
```

Comment:
Do we need add task name in the personal working branch name, for example 'wip_<your-github-id>_<task-name>'?

Reply:
OK. I'd add this format as a new optional naming convention for personal working branches.